manifests/overlays/standalone/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Do we want to name this overlay as `default` to be consistent with other projects (e.g. JobSet, Kueue)?